msa/js-executor/queue/kafkaTemplate.js

```diff
@@ -197,6 +284,10 @@ async function disconnectProducer() {
         var _producer = producer;
         producer = null;
         try {
+            logger.info('Stopping loop...');
+            //TODO: send handle msg
```

Comment:
What is this about?

Reply:
we have to await sendMessagesAsBatch to send all ready-to-send messages before close gracefully. 
In actual implementation, the worst scenario that can happen on tb-node is a timeout and resend the JS message to process again.
I'll consult with @vvlladd28 on how to resolve this TODO.